plugins/woocommerce-blocks/assets/js/base/components/combobox/style.scss

```diff
@@ -1,3 +1,10 @@
+// For when the Combobox is being used as a select custom field.
+.wc-block-components-select-input {
+	position: relative;
+	margin-top: $gap;
+	white-space: nowrap;
```

Comment:
I copied this from the text input, so that we keep the styling consistent with the other inputs on our checkout:

Their CSS is here:

https://github.com/woocommerce/woocommerce/blob/29b4217f154f967190eb27914a122937d3a992fb/plugins/woocommerce-blocks/packages/components/text-input/style.scss#L2

Did you try entering text longer than the input field to see if it is a problem?

Reply:
Ok thanks for spotting this Niels, you're right! Change added in 7881ec5435c7be2739690e8268b3e4447c4b0b1d